config/repos.yaml

```diff
@@ -176,6 +176,7 @@ Katello/foreman-gutterball:
 Katello/foreman_sam:
   redmine: katello
 Katello/hammer-cli-csv:
+  pr_scanner: true
```

Comment:
The project doesn't seem to use Jenkins for pull requests, are you sure you need this enabled?